.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,116 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
Is it intended to commit this file?

Reply:
Android studio keep adding it every time i remove it, I guess it's a file needed by the latest update of AS

Reply:
OK